py/torch_tensorrt/dynamo/lowering/_partition.py

```diff
@@ -85,7 +90,11 @@ def propose_partitions(self) -> List[Partition]:
                 ):
                     compute_node_count += 1
 
-            if compute_node_count < self.min_block_size and not exempted_partition:
+            if (
+                compute_node_count < self.min_block_size
+                and not exempted_partition
+                and not full_support
+            ):
```

Comment:
Alternatively, if preferable, we can display a warning in such a case, but still respect the minimum block size argument